### PR TITLE
Warning: NLS missing message: MavenTargetLocationWizard_11 in: org.eclipse.m2e.pde.ui.editor.messages

### LINKED ISSUE #580: Warning: NLS missing message: MavenTargetLocationWizard_11 in: org.eclipse.m2e.pde.ui.editor.messages
I see at startup the following message cc @laeubi while using latest snapshot

Warning: NLS missing message: MavenTargetLocationWizard_11 in: org.eclipse.m2e.pde.ui.editor.messages

### PR DESCRIPTION
**Pull request by vogella**
Fixes #580

### CODE REVIEW

**Comment by mickaelistria** [comment]
Please bump versions where necessary.

**Comment by mickaelistria** [comment]
Bump missing in the pde.feature.

**Comment by vogella** [comment]
Feature version also increased

File: org.eclipse.m2e.pde.feature/feature.xml [hunk 1]
```diff
@@ -1,63 +1,7 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <feature
-      id="org.eclipse.m2e.pde.feature"
-      label="%featureName"
-      version="1.20.0.qualifier"
-      provider-name="%providerName"
-      plugin="org.eclipse.m2e.core"
+      version="1.20.100.qualifier"
```
Comment (mickaelistria) [inline org.eclipse.m2e.pde.feature/feature.xml:3]: Some important attributes and other content have vanished.

**Review by mickaelistria — CHANGES_REQUESTED** [review]

File: org.eclipse.m2e.pde.feature/feature.xml (hunk 1 above)
Comment (vogella) [reply org.eclipse.m2e.pde.feature/feature.xml:5]: Sorry, fixed and force pushed to update this PR.

**Pull request merged**